src/schema/v2/partner.ts

```diff
@@ -79,6 +81,44 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
     return {
       ...SlugAndInternalIDFields,
       cached,
+      articlesConnection: {
+        description: "A connection of articles related to a partner.",
+        type: articleConnection.connectionType,
+        args: pageable({ sort: ArticleSorts }),
+        resolve: async ({ _id }, args, { articlesLoader }) => {
+          const { size, offset, sort } = convertConnectionArgsToGravityArgs(
+            args
+          )
+
+          interface GravityArgs {
```

Comment:
Since we're actually querying a different backend (not "Gravity", which is our main API) with this request, can we rename this interface to just `Args` or `articleArgs` or something?